cmake/CMakeLists.txt

```diff
@@ -12,8 +12,11 @@ set(CMAKE_POLICY_DEFAULT_CMP0069 NEW)
 include(CheckCXXCompilerFlag)
 include(CheckLanguage)
 
-# Set C++11 as standard for the whole project
-set(CMAKE_CXX_STANDARD 11)
+# CentOS compiler is old but it does allow certain C++14 features
+# such as lambda captures and they are convinient
```

Comment:
# g++ --std=c++14
g++: error: unrecognized command line option '--std=c++14'

Reply:
This is actually the original code, not my invention. The option has one leading dash.